recipes/R/Retail - GradientBoosting/R/applicationEvaluator.R

```diff
@@ -38,13 +38,14 @@ applicationEvaluator <- setRefClass("applicationEvaluator",
                                                             user_token = configurationJSON$ML_FRAMEWORK_IMS_TOKEN,
                                                             service_token = configurationJSON$ML_FRAMEWORK_IMS_ML_TOKEN)
       
-      df <- reader$load(configurationJSON$trainingDataSetId, configurationJSON$ML_FRAMEWORK_IMS_ORG_ID)
+      df <- reader$load(configurationJSON$dataSetId, configurationJSON$ML_FRAMEWORK_IMS_ORG_ID, batch_id = configurationJSON$batchId)
```

Comment:
Isn't the second filter() in the else condition redundant?

Reply:
Yeah, I made a mistake here. Will correct immediately.